templates/items/details/details-uses.hbs

```diff
@@ -0,0 +1,72 @@
+<fieldset>
+    <legend>{{ localize "DND5E.Usage" }}</legend>
+
+    {{!-- Uses --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.LimitedUses" }}</label>
+        <div class="form-fields">
+
+            {{!-- Current --}}
+            <div class="form-group label-top">
+                <label>{{ localize "DND5E.Current" }}</label>
+                <div class="form-fields">
+                    <input type="number" name="system.uses.value" value="{{ system.uses.value }}" min="0" step="1"
+                           placeholder="0">
+                </div>
+            </div>
+
+            {{!-- Max --}}
+            {{ formField fields.uses.fields.max value=source.uses.max label="DND5E.Max" localize=true hint=""
+                         classes="label-top" }}
+        </div>
+    </div>
+
+    {{!-- Auto-Destroy --}}
+    {{#if fields.uses.fields.autoDestroy}}
+    {{ formField fields.uses.fields.autoDestroy value=source.uses.autoDestroy localize=true
+                 input=inputs.createCheckboxInput }}
+    {{/if}}
+</fieldset>
+
+{{#if system.hasLimitedUses}}
```

Comment:
This means recovery isn't visible on non-embedded items if they reference an actor property in the max formula (e.g. `@prof`), because that resolves to `0` during data preparation. We'll probably want to tweak `ActivitiesTemplate#hasLimitedUses` to return `true` for non-embedded items if they have any max uses formula in the source data.

Reply:
I would think checking for a non-empty string is probably good enough for embedded and non-embedded cases. Presumably if you're filling in *some* value, you're expecting that it might have recovery profiles, even if it happens to evaluate to `0` currently.